release-please-config.json

```diff
@@ -0,0 +1,13 @@
+{
+  "packages": {
+    "$schema": "https://raw.githubusercontent.com/googleapis/release-please/main/schemas/config.json",
+    "release-type": "node",
```

Comment:
Can you remove `release-type` from the `npm-publish` workflow? Then we should be good to go.